pages/contact.js

```diff
@@ -12,28 +12,116 @@ export default class Contact extends Component {
         <Helmet
           title={config.siteTitle}
         />
-        <div className="contact--center">
-
-          <div className="contact--title">
+      
+        <div className="page__title">
             Contact
-          </div>
-
-          <a href="#" target="_blank">
-            <FontAwesome
-              className="media--icon fb"
-              name="facebook"
-            /> 
-          </a>
-
-          <a href="#" target="_blank">
-            <FontAwesome
-              className="media--icon mail"
-              name="envelope"
-            /> 
-          </a> 
+        </div>
+
+        <div className="page__content">
+          <div className="contact__name">International Relations Council </div>
```

Comment:
Can you add a wrapper around ```contact__name``` and ```contact__icons``` and then add margins around the wrapper so everything is more spaced out?

```scss
.contact__wrapper {
  // margins

  .contact__name {
    //styles  
  }
 .contact__icons {
    //styles  
  }
}
```